src/extension/background-script/index.ts

```diff
@@ -12,6 +12,8 @@ import state from "./state";
 let isFirstInstalled = false;
 let isRecentlyUpdated = false;
 
+const debug = process.env.NODE_ENV || false;
```

Comment:
Is `NODE_ENV` is also set when we [do the builds](https://github.com/getAlby/lightning-browser-extension/blob/master/package.json#L20-L22)? Then this will also show logs in prod?